sirmordred/task_collection.py

```diff
@@ -74,7 +75,7 @@ def execute(self):
             logging.info('%s collect disabled', self.backend_section)
             return errors
 
-        t2 = time.time()
+        time_start = datetime.now()
```

Comment:
Shouldn't be better to use the [datetime_utcnow()](https://github.com/chaoss/grimoirelab-toolkit/blob/master/grimoirelab_toolkit/datetime.py#L62) from `grimoirelab_toolkit`?

Reply:
The enrichment was using `datetime.datetime.now()` to calculate the spent time, that's why I used the same. I updated it to use `datetime_utcnow()`, should I update the enrichment to use `datetime_utcnow()`?